examples/green-house-migration/routes/api/metadata.py

```
@@ -0,0 +1,119 @@
+
+from fastapi import APIRouter, Depends
+
+from legacy.greenhouse.client import gh_get, paginated_get
+from legacy.greenhouse.pagination import pagination_dependency
+
+router = APIRouter(prefix="/metadata", tags=["Metadata"])
+
+
+@router.get("/cthee_reaares")
+def list_cthee_reaares(
+    pagination: Dict[str, Any] = Depends(pagination_dependency),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)